installer/centos/README.md

```diff
@@ -5,14 +5,14 @@ The script below assumes that it can update OS packages for python and prerequis
 * CentOS 7: [centos7-installer.sh](centos7-installer.sh)
 
     ```bash
-    curl --remote-name https://raw.githubusercontent.com/jonescompneurolab/hnn/master/installer/ubuntu/hnn-centos-7.sh
```

Comment:
I hope this didn't take too long to spot :smile:

Reply:
Longer than it should have... :cry: